projects/use/useop-qwt-config.cmake

```diff
@@ -0,0 +1,28 @@
+# Get the path to the overseer pro install
+get_filename_component(OP_ROOTDIR ${CMAKE_CURRENT_LIST_DIR}/../.. ABSOLUTE)
+get_filename_component(OP_ROOTDIR ${OP_ROOTDIR} ABSOLUTE)
+
+unset(QWT_INCLUDE_DIR CACHE)
+unset(QWT_LIBRARY_DIR CACHE)
+
+set(QWT_INCLUDE_DIR ${OP_ROOTDIR}/include/qwt)
+set(QWT_LIB_DIR ${OP_ROOTDIR}/lib)
+
+# determine OP_BUILD_STATIC
+include(${CMAKE_CURRENT_LIST_DIR}/opopts.cmake)
+
+string(TOLOWER ${CMAKE_BUILD_TYPE} build_type)
+
+if(WIN32)  
+  if(${build_type} STREQUAL debug)
+    set(QWT_LIB ${QWT_LIB_DIR}/qwtd.lib)
+  else()
+    set(QWT_LIB ${QWT_LIB_DIR}/qwt.lib)
+  endif()
+else()
+  if(OP_BUILD_STATIC)
```

Comment:
If you look in the qwt.cmake you'll see where I'm passing in STATIC_BUILD=true vs. STATIC_BUILD=false.  I added those as options to be able to make it build static vs. dynamic.